cli/operator/node.go

```diff
@@ -736,68 +716,57 @@ func setupEventHandling(
 		fromBlock = new(big.Int).SetUint64(fromBlock.Uint64() + 1)
 	}
 
-	// load & parse local events yaml if exists, otherwise sync from contract
-	if len(cfg.LocalEventsPath) != 0 {
-		localEvents, err := localevents.Load(cfg.LocalEventsPath)
```

Comment:
https://github.com/ssvlabs/ssv/pull/1819/files#r1826980844 `cfg.LocalEventsPath` is used here